steam_wrapper/steam_wrapper.py

```diff
@@ -351,6 +352,11 @@ def configure_shared_library_guard():
     else:
         os.environ["LD_AUDIT"] = f"/app/links/$LIB/libshared-library-guard.so"
 
+def clear_empty_environment(current_info):
+    environment = current_info["environment"]
+    for key, value in os.environ.items():
+        if environment.get(key.lower()) == "":
```

Comment:
Why convert to lowercase? I believe environment variables are case-sensitive and e.g. `TZ` and `tz` are actually different variables.

Reply:
Python configparser quirk. Keys are lowercase.

Reply:
Ah, right. I think we better fix that instead. Probably set `flatpak_info.optionxform = str` in `read_flatpak_info()` before loading the file would be enough?